qadence/constructors/hamiltonians.py

```diff
@@ -236,23 +235,11 @@ def _update_interaction_strength(
     return register
 
 
-# FIXME: Previous hamiltonian / observable functions, now refactored, to be deprecated:
-
-DEPRECATION_MESSAGE = "This function will be removed in the future. "
-
-
 def single_z(qubit: int = 0, z_coefficient: float = 1.0) -> AbstractBlock:
-    message = DEPRECATION_MESSAGE + "Please use `z_coefficient * Z(qubit)` directly."
-    warnings.warn(message, FutureWarning)
     return Z(qubit) * z_coefficient
 
```

Comment:
does it mean those functions won't be deprecated anymore?

Reply:
yes